admin/autoupdater/src/autoupdater.c

```diff
@@ -334,6 +343,17 @@ static bool autoupdate(const char *mirror, struct settings *s, int lock_fd) {
 		goto out;
 	}
 
+	if (s->check) {
+		printf(
+			"VERSION=%s\n"
```

Comment:
Hmm, do we want to add single quotes around the version number, so the output becomes parsable as shell code even when weird characters appear in the version (excluding the single quote...)?